travis-ymls/boto.travis.yml

```diff
@@ -0,0 +1,92 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : boto
+# Source Repo         : https://github.com/boto/boto/
+# Travis Job Link     : https://travis-ci.com/github/asellappen/boto/
+# Created travis.yml  : No
+# Maintainer          : Arumugam N S <asellappen@yahoo.com>/Priya Seth<sethp@us.ibm.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+language: python
+
+# Some versions of python only work on certain platforms. Specifying the distribution
+# to ensure the version of python will run without issue.
+matrix:
+  include:
+    # Trusty Tests
+    - os: linux
+      dist: trusty
+      python: "pypy"
+    - os: linux
+      dist: trusty
+      python: "2.6"
+    - os: linux
+      dist: trusty
+      python: "3.3"
+    - os: linux
+      dist: trusty
+      python: "3.4"
+    # Xenial Tests
+    - os: linux
+      dist: xenial
+      python: "2.7"
+    - os: linux
+      dist: xenial
+      python: "3.5"
+    - os: linux
+      dist: xenial
+      python: "3.6"
+    - os: linux
+      dist: xenial
+      python: "3.7"
+    - os: linux
+      dist: xenial
+      python: "pypy3.5"
+      env: 
+       - export CRYPTOGRAPHY_ALLOW_OPENSSL_102=1
+    # bionic Tests
```

Comment:
Are we adding this new job for intel ?

Reply:
no,it's existing job only based on devlope branch (https://github.com/boto/boto/blob/develop/.travis.yml)

Reply:
If you are picking develop branch instead of master branch for any specific reason, pl. mention the same as part of comment

Reply:
From what I can see in the "develop" branch this target for intel was not existing and you have added it newly

Reply:
wanted to cover bionic also in the test part since ppc supporting from Xenial only.
amd64 covered 
Trust,Xenial and bionic
ppc64le covered 
Xenial and bionic.